LayoutTests/platform/mac-bigsur/tables/mozilla/bugs/bug2479-3-expected.txt

```diff
@@ -85,7 +85,7 @@ layer at (0,0) size 785x765
             RenderTextControl {INPUT} at (402,16) size 147x19 [bgcolor=#FFFFFF] [border: (2px inset #000000)]
           RenderText {#text} at (548,17) size 17x16
             text run at (548,17) width 17: " "
-          RenderButton {INPUT} at (564,17) size 54x18 [color=#000000D8] [bgcolor=#C0C0C0]
+          RenderButton {INPUT} at (564,17) size 54x18 [color=#000000D8] [bgcolor=#C0C0C0] [border: none (2px outset #C0C0C0) none (2px outset #C0C0C0)]
```

Comment:
Where does the border come from here? Given that the size hasn't changed, I don't think there should be rendering differences?

Reply:
There's some 2px spacing on the left/right side that now appears (top/bottom is `none` like the dump shows), similar to the one on `<button>` currently, but it's not visible anyway since the native appearance.